### ISSUE TITLE
Missing typing_extensions dependency in setup.py

### ISSUE THREAD

**Issue by velovix** [elarivie/pyReaderWriterLock#4]
When installing this library with pip in a fresh environment, importing will result in a `ModuleNotFoundError`.

```
>>> from readerwriterlock import rwlock
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/velovix/.virtualenvs/testing/lib/python3.6/site-packages/readerwriterlock/rwlock.py", line 12, in <module>
    from typing_extensions import Protocol
ModuleNotFoundError: No module named 'typing_extensions'
```

Unsurprisingly, installing `typing_extensions` manually solves the problem, but I think this should be included in the setup.py for the project so it's installed automatically.

**Comment by elarivie** [comment]
@velovix 
Oups my bad 😞

Thanks for your issue report

It is now fix in v1.0.6

**Comment by velovix** [comment]
Thanks for the quick response!
